package.json

```diff
@@ -14,7 +14,12 @@
     "watch": "npm run build -- --watch",
     "start": "webpack-dev-server --config examples/using-module/webpack.config.babel.js --context examples/using-module --open",
     "analyze": "webpack --profile --json --progress > stats.json",
-    "lint": "eslint . --cache && stylelint **/*.scss --cache"
+    "lint-js": "eslint . --cache",
+    "lint-sass": "stylelint **/*.scss --cache",
+    "lint": "yarn lint-js && yarn lint-sass",
+    "fix-js": "yarn lint-js --fix",
```

Comment:
When would we want to run these commands? Is this for anyone who doesn't have the VSCode plugin?

Reply:
I run this from time to time. Sometimes my linter plugin acts up and I'd rather finish a feature, run this command, then figure out what's wrong with my tooling while my PR is being reviewed. It happens more than I'd like, but it's nice to have a quick backup to stay in the flow.